.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file comes from using VSCode, which I switched to at PyCascades and am LOVING so far. It's not super relevant to this PR.